src/App.js

```diff
@@ -5,16 +5,48 @@ import Button from "@mui/material/Button";
 import CssBaseline from "@mui/material/CssBaseline";
 import Box from "@mui/material/Box";
```

Comment:
imports for `Box` and `Button` are unused. It is preferred to remove those imports.

Reply:
done